database/migrations/2020_12_26_154715_add_recat_category_to_aircraft.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddRecatCategoryToAircraft extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('aircraft', function (Blueprint $table) {
+            $table->unsignedBigInteger('recat_category_id')
+                ->nullable()
+                ->after('wake_category_id')
+                ->comment('The aircrafts RECAT category');
+
+            $table->foreign('recat_category_id')->references('id')->on('recat_categories');
```

Comment:
Do you want any cascade behaviour?

Reply:
Not on this one I dont think - if a RECAT category changes we want to recategorize, rather than drop rows